src/behavior.js

```diff
@@ -18,7 +18,7 @@ var Behavior = MNObject.extend({
     // wants to directly talk up the chain
     // to the view.
     this.view = view;
-    this.defaults = _.result(this, 'defaults') || {};
+    this.defaults = _.clone(_.result(this, 'defaults', {}));
```

Comment:
Well.. so do we want to update the underscore dep to 1.8?  1.7 doesn't have a default value for `_.result`  Not sure about when lodash added it either.